tincr/cad/util/util.tcl

```diff
@@ -328,6 +329,55 @@ proc ::tincr::print { args } {
     }
 }
 
+proc ::tincr::print_list { args } {
+    # Summary:
+    # Prints a list to specified channel with the specified header
+
+    # Argument Usage:
+    # [-newline] : Add a new line character after printing each element. 
+    #              If this is not specified, a space will be printed between each element 
+    # -channel : The output channel to print to
+    # -header : The header to print before the list (i.e. HEADER item1 item 2)
+    # print_list : The list to print
+
+    # Return Value: none
+
+    # Categories: xilinxtclstore, byu, tincr, util
+
+    # Notes:
+    # The default channel is "stdout".
+    
+    # Usage:
+    # tincr::print_list [-channel arg] [-header arg] [-newline] print_list
```

Comment:
This format of commenting a method (method comment inside of the method, format, etc.) is obsolete. Wyatt and I were using it because it's the format the Xilinx Tcl store requires. Since we decided to host Tincr on Github instead of the Xilinx Tcl store, it's better to follow Doxygen's format: https://www.stack.nl/~dimitri/doxygen/manual/docblocks.html#tclblocks